ai-ml/trainium-inferentia/eks.tf

```diff
@@ -507,30 +557,31 @@ module "eks" {
         export PATH=/opt/aws/neuron/bin:$PATH
       EOT
 
-      min_size     = 0
-      max_size     = 2
-      desired_size = 0
+      min_size     = var.inf2_24xl_min_size
+      max_size     = var.inf2_24xl_max_size
```

Comment:
max_size is hardcoded now in recent changes of the main. pull the latest code from main branch

Reply:
Got it.